src/annotator/integrations/vitalsource.ts

```diff
@@ -22,11 +22,41 @@ import type { InjectConfig } from '../hypothesis-injector';
 // smaller and it feels unreadable or too-zoomed-out
 const MIN_CONTENT_WIDTH = 480;
 
+/**
+ * Book metadata exposed by the VitalSource viewer.
+ */
+type BookInfo = {
```

Comment:
While this type is private to the `vitalsource` integration, I wonder if it might be helpful to name it `VitalSourceBookInfo` for clarity and grep-ability.  Take or leave this suggestion.